src/components/dialog/Dialog.spec.tsx

```diff
@@ -1,181 +1,322 @@
 import * as React from 'react';
-import {mount} from 'enzyme';
 import Dialog from './Dialog';
 import DialogCloseButton from './DialogCloseButton';
+import {
+  render,
+  fireEvent,
+  waitForElementToBeRemoved,
+  screen,
+} from '@testing-library/react';
+import {waitFor} from '@storybook/testing-library';
+import userEvent from '@testing-library/user-event';
+import {testA11y} from '../../axe';
+import DialogBody from './DialogBody';
+import DialogHeader from './DialogHeader';
 
 window.scrollTo = jest.fn();
+
 describe('<Dialog>', () => {
   it('renders children', () => {
-    const wrapper = mount(<Dialog open>content text</Dialog>);
+    const wrapper = render(<Dialog open>content text</Dialog>);
 
-    expect(wrapper.contains('content text')).toBe(true);
+    expect(wrapper.getByText('content text')).toBeTruthy();
   });
+
   it('renders proper size', () => {
```

Comment:
@coderitual @clxandstuff @domidomi Maybe we should remove tests like these (purely visual and with stories)? I think relying on chromatic is enough